Lil-Peep/src/styles/modules/site-collection.scss

```diff
@@ -0,0 +1,295 @@
+  display: flex;
+  justify-content: flex-end;
+  padding-left: 30px;
+  padding-right: 30px;
+
+  @media #{$breakpoint-mobile} {
+    justify-content: space-between;
+  }
+
+  .active-button {
+    color: #636e72;
+    display: inline;
+    font-size: 30px;
+    text-decoration: none;
+
+    @media #{$breakpoint-mobile} {
+      font-size: 20px;
+    }
+  }
+
+  .blocked-button {
+    color: #636e72;
+    display: inline;
+    font-size: 30px;
+    opacity: 0.5;
+    text-decoration: none;
+
+    @media #{$breakpoint-mobile} {
+      font-size: 20px;
+    }
+  }
+
+  .previous {
+    margin-right: 50px;
+    @media #{$breakpoint-mobile} {
+      margin-right: 0;
+    }
+  }
+}
+
+  .collection-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .product-card-category {
+    background-color: #eaebec;
+    flex: 1 0 20%;
+    height: 470px;
+    margin: 10px;
+    max-width: 435px;
+
+    &:hover {
+      background-color: rgba(83, 150, 230, 0.1);
+    }
+
+    .front-label {
+      display: flex;
+      justify-content: flex-end;
+      width: 100%;
+
+      .new-product {
+        background-color: #abc828;
+        border-radius: 3px;
+        color: #ffffff;
+        font-weight: bold;
+        padding: 5px 25px;
+      }
+
+      .not-new-product {
+        color: #eaebec;
+        opacity: 1;
+        padding: 5px 25px;
+      }
+    }
+
+    .product-image {
+      align-items: center;
+      display: flex;
+      flex-direction: row;
+      height: 300px;
+      justify-content: center;
+    }
+
+    img {
+        height: auto;
+        max-height: 300px;
+        max-width: 250px;
+        object-fit: contain;
+      }
+  }
+
+.product-data {
+  padding-left: 15px;
+  padding-right: 15px;
+
+  p {
+    font-family: 'Lato', sans-serif;
+    margin-bottom: 5px;
+    margin-top: 0;
+  }
+
+  .product-title {
+    color: #636e72;
+    font-weight: bold;
+    text-decoration: none;
+  }
+
+  .price {
+    color: #954180;
+  }
+
+  .offer {
+    display: none;
+  }
+
+  .view-in-detail {
+    background-color: #4a90e2;
+    border-radius: 3px;
+    border-style: none;
+    padding: 10px 0;
+    text-align: center;
+    width: 100%;
+
+    &:hover {
+      background-color: #ed7f03;
+      cursor: pointer;
+    }
+
+    a {
+      text-decoration: none;
+    }
+
+    h4 {
+      background-color: #4a90e2;
+      color: #ffffff;
+      font-family: 'Oswald', sans-serif;
+      font-weight: bold;
+      margin: 0;
+    }
+  }
+
+  .price {
+    @media #{$breakpoint-tablet} {
+      display: flex;
+    }
+  }
+
+  .offer {
+    @media #{$breakpoint-tablet} {
+      color: #D2D2D2;
+      display: flex;
+      margin-left: 5px;
+      text-decoration: line-through;
+    }
+  }
+}
+
+.scroll-up {
+  align-items: center;
+  background-color: #387ada;
+  border-radius: 3px;
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  margin-bottom: 20px;
+  margin-top: 20px;
+  padding: 0 10px 0 10px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 10px` instead of `0 10px 0 10px`